.travis.yml

```diff
@@ -36,6 +36,11 @@ env:
     - BUILD_PROCESSOR="`uname -p`"
     - OPENMVG_SOURCE=${TRAVIS_BUILD_DIR}/src
     - OPENMVG_BUILD=${TRAVIS_BUILD_DIR}/build
+    # CMAKE
+    - CMAKE_URL="https://cmake.org/files/v3.4/cmake-3.4.1-Linux-x86_64.tar.gz"
```

Comment:
https://cmake.org/files/v3.0/cmake-3.0.0.tar.gz in case we want to use the minimum version